scenarios/langchain/getting-started-with-langchain-embeddings.ipynb

```diff
@@ -0,0 +1,167 @@
+{
+ "cells": [
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "# LangChain and Azure AI Foundry\n",
+    "\n",
+    "This notebook explain how to use `langchain-azure-ai` package with the capabilities in Azure AI Foundry."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 1. Prerequisites\n",
+    "\n",
+    "To run this tutorial you need either:\n",
+    "\n",
+    "1. Using GitHub Models:\n",
+    "\n",
+    "    1. You can use [GitHub models](https://github.com/marketplace/models) endpoint including the free tier experience.\n",
+    "    2. Use the endpoint `https://models.inference.ai.azure.com` along with your GitHub Token.\n",
+    "\n",
+    "1. Using Azure AI Foundry:\n",
+    "\n",
+    "    1. Create an [Azure subscription](https://azure.microsoft.com).\n",
+    "    2. Create an Azure AI hub resource as explained at [How to create and manage an Azure AI Studio hub](https://learn.microsoft.com/en-us/azure/ai-studio/how-to/create-azure-ai-resource).\n",
+    "    3. Deploy one model supporting the [Azure AI model inference API](https://aka.ms/azureai/modelinference). In this example we use a `text-embedding-3-large` deployment. \n",
+    "\n",
+    "        * You can follow the instructions at [Add and configure models to Azure AI model inference service](https://learn.microsoft.com/azure/ai-studio/ai-services/how-to/create-model-deployments)."
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "Install the following packages:\n",
+    "\n",
+    "```bash\n",
+    "pip install -U langchain-core langchain-azure-ai\n",
+    "```"
+   ]
+  },
+  {
+   "cell_type": "markdown",
+   "metadata": {},
+   "source": [
+    "## 2. Use embeddings models\n",
+    "\n",
+    "Create a client to connect to the endpoint. In this case, we are working with an embeddings model hence we import the class `AzureAIEmbeddingsModel`."
+   ]
+  },
+  {
+   "cell_type": "code",
+   "execution_count": null,
+   "metadata": {},
+   "outputs": [],
+   "source": [
+    "import os\n",
+    "from langchain_azure_ai.embeddings import AzureAIEmbeddingsModel\n",
+    "\n",
+    "embed_model = AzureAIEmbeddingsModel(\n",
+    "    endpoint=os.environ[\"AZURE_INFERENCE_ENDPOINT\"],\n",
```

Comment:
should we have a version of this that just takes a project connection string and model name? that way users don't need to store the inference endpoint/credential in plain text

Reply:
Let's do this next release!